readme.md

```diff
@@ -110,7 +110,7 @@ double function(int i, double j)
   return i * j;
 }
 
-int main()
+int main();
```

Comment:
Don't think this ; is needed.

Reply:
Probably a result of https://hacktoberfest.digitalocean.com/